README.md

```diff
@@ -7,6 +7,9 @@ An implementation of a symbolic toolbox using SymPy.
 
 [https://octave.sourceforge.io/symbolic]
 
+<sub>Note: Development for most Octave packages tends to take place in sourceforge. However, unlike most other Octave packages, the main development repository for this package is on **github** (https://github.com/cbm755/octsympy), and only _mirrored_ on sourceforge (https://sourceforge.net/p/octave/symbolic/).
+If you are thinking of contributing and you're looking at this README file from the sourceforge website, please consider visiting our github page and contributing there instead, as we do not tend to monitor sourceforge for activity.</sub>
+
```

Comment:
Please keep these changes on topic for this PR. These README suggestions should be made on a separate branch and PR.

Reply:
Hi Mike, yes, I pushed to my fork hoping to create a completely separate pull request, and it automatically got merged into this one for some reason because it was an already active pull request from my repository (as I understand it).

Do you know how to create a separate pull request by any chance? Or is the only way to wait until this pull request has been dealt with by the maintainer first?

Reply:
(or how to revert this particular pull request so that it only reflects the cumsum/cumprod commit?)

Reply:
Yeah, a branch is linked to a PR, so any new commits on that branch are part of the same PR. You pushed changes to your "master" branch, I typically use a custom branch name for each PR.